src/pages/ResearcherProfile.js

```diff
@@ -378,8 +378,9 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
         }
       } else {
         this.saveUser().then(resp => {
-          this.setState({ showDialogSubmit: false });
-          this.props.history.goBack();
+          this.setState({ isResearcher: resp.isResearcher, showDialogSubmit: false });
+          //Should the code redirect the user to the Researcher console or should it redirect to the expanded profile on account creation
+          //QUESTION: Is there a reason why the expanded profile page isn't the default view?
```

Comment:
I've asked this question before too and Jonathan wants a successfully registered user to be directed to the catalog (or the default for a researcher which is NOT their console) where they can immediately request a DAR.